bytestring/ppx.t/run.t

```diff
@@ -92,7 +94,8 @@
          
          Valid sizes are:
          
-           bytes       - match on the entire string
+           bytes      - match on the literal string
+           string       - match on the entire string
```

Comment:
**fix**. Super small but these should be aligned so when the error pops up for the user they look clean/easy to visually parse.